docs/technical_reference/unit_models/steam_ejector.rst

```diff
@@ -0,0 +1,69 @@
+Steam Jet Ejector
+================================
+This Steam Jet Ejector unit model:
+   * Simulates the performance of a steam jet ejector for thermal vapor compression.
+   * Uses semi-empirical correlations for entrainment ratio, PCF, and TCF based on El-Dessouky (1997).
+   * Operates in steady-state only.
+   * Assumes the discharge mixture pressure equals its saturation pressure.
+   
+.. index::
+   pair: watertap.unit_models.steam_ejector;steam_ejector
+
+.. currentmodule:: watertap.unit_models.steam_ejector
+
+Degrees of Freedom
+-------------------
+In addition to the inlet state variables (i.e., temperature, pressure, and component flowrates for motive steam and entrained vapor), the Steam Ejector model has at least 1 degrees of freedom that must be fixed for the unit to be fully specified. Typically, the following variables are fixed:
+
+    * Entrainment ratio
+    * Compression ratio
+
+Model Structure
+------------------
+This Steam Ejector model consists of state blocks for the properties of the motive steam inlet, entrained vapor inlet, and discharge mixture. It incorporates semi-empirical equations to model key performance parameters.
+
+Equations
+---------
```

Comment:
I would arrange these sections in the following order: Sets -> Variables -> Performance Metrics -> Equations. That's been the convention with unit model documentation so far and it makes more sense to spell out variable names before invoking them in the equations

Reply:
addressed